tautulli.json

```diff
@@ -1,12 +1,8 @@
 {
     "name": "Tautulli",
     "release": "12.2-RELEASE",
-    "artifact": "https://github.com/arcooke/iocage-plugin-tautulli.git",
+    "artifact": "https://github.com/dropsignal/iocage-plugin-tautulli.git",
     "official": false,
-    "properties": {
-        "nat": 1,
```

Comment:
This plugin should not default to using NAT anymore ?

Reply:
I've renamed post-install.sh to post_install.sh. https://github.com/arcooke/ returns a 404 error and the plugin install fails. I've recreated the plugin. I do not use NAT so I changed the default to my preference.